css/index.css

```diff
@@ -1,3 +1,154 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+* {
+	box-sizing: border-box;
+	font-family: "Titillium Web", sans-serif;
+}
+body {
+	margin: 2vh 8vh;
```

Comment:
I would suggest only putting global styles for font in the body selector.  A wrapper div with a class container can be used to wrap everything in the body and apply a percentage width and margin 0, auto